### PR TITLE
Update migrating-telemetry-and-security-agents.md

### PR DESCRIPTION
**Pull request by ptux**
Make the code snippet clear.

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by wiggitywhitney** [comment]
/lgtm

\(^-^)/

**Comment by wiggitywhitney** [comment]
@bradtopol I reviewed this and it looks great to me as well

File: content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ For example: on COS images, Docker exposes its Unix domain socket at
 
 Here's a sample shell script to find Pods that have a mount directly mapping the
 Docker socket. This script outputs the namespace and name of the pod. You can
-remove the grep `/var/run/docker.sock` to review other mounts.
+remove the `grep '/var/run/docker.sock'` to review other mounts.
```
Comment (wiggitywhitney) [inline content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:55]: @bradtopol I reviewed this and it looks great to me as well

**Review by bradtopol — COMMENTED** [review]
looks great!

**Pull request merged**